alfasim_sdk/types.py

```diff
@@ -58,12 +70,20 @@ class String(BaseField):
 
 
     """
-
     value: str = attrib(validator=[instance_of(str), check_string_is_not_empty])
 
 
 @attr.s(kw_only=True)
 class Enum(BaseField):
+    """
+    The Enum field provides list of options to the user, showing  only the select item but providing a way to display
+    a list of all options through a combo-box.
+
+        :ivar str caption: A string to be displayed on the right side of the component.
```

Comment:
```suggestion
    :ivar str caption: A string to be displayed on the right side of the component.
```

Reply:
You cannot indent it like this otherwise RST will interpret this as unformated text.